### ISSUE TITLE
Question: removing hashtag in url Index page?

### ISSUE THREAD

**Issue by valeriosillari** [nuxt/nuxt.js#2362]
I would like to know if it possible remove the hashtag in the index page.

I see it in dev and live deployed pages:

localhost:3000/#/
myProjectDeployedLive.com/#/

I read the documentation here and looks related to the router logic.
https://nuxtjs.org/api/configuration-router/

As default the option should be "mode: history" and on Vue this should solve the problem.
But looks not working here on Nuxt.

Any suggestions or explanation?

**Comment by clarkdo** [comment]
Actually the default value of `route.mode` is `history` and there is no `#` in url.

Can you confirm if you config `nuxt.config.js` correctly or provide a repo can reproduce you issue?

**Comment by valeriosillari** [comment]
Problem solved. In a plugins for goggle analytics I called the vue router and this create the issue. 
removed 2 lines of code in the plugin, everything is fine.

FYI: here the link at the vue plugin, to the tracking option where I set the vue router following the info. I am quite sure I made some mistakes here following the documentation ... ☺️ 
https://github.com/MatteoGabriele/vue-analytics/blob/master/docs/page-tracking.md#enable-page-auto-tracking
